src/core/client/admin/components/UserHistoryDrawer/UserHistoryDrawerQuery.tsx

```diff
@@ -78,20 +78,10 @@ const UserHistoryDrawerQuery: FunctionComponent<Props> = ({
               <span>{user.username}</span>
             </Flex>
             <div className={styles.userStatus}>
-              <Flex alignItems="center">
-                <div className={styles.userStatusLabel}>
-                  <Typography variant="bodyCopyBold" container="div">
-                    <Flex alignItems="center" itemGutter="half">
-                      <Localized id="moderate-user-drawer-status-label">
-                        Status:
-                      </Localized>
-                    </Flex>
-                  </Typography>
-                </div>
-                <div className={styles.userStatusChange}>
-                  <UserStatusChangeContainer user={user} fullWidth={false} />
-                </div>
-              </Flex>
+              <div className={styles.userStatusLabel}>Status:</div>
+              <div className={styles.userStatusChange}>
+                <UserStatusChangeContainer user={user} fullWidth={false} />
```

Comment:
Prefer a `fullWidth` that defaults to false rather than needing to specify `fullWidth={false}`